api-server/migrations/2021-05-27-195835_expand-db/down.sql

```diff
@@ -1,3 +1,3 @@
-DROP TABLE sensors
-DROP TABLE measurements
-DROP TABLE sensor_types
\ No newline at end of file
+DROP TABLE sensors;
+DROP TABLE measurements;
+DROP TABLE sensor_types;
```

Comment:
Haha "Depython'd". If that's what adding semicolons is then I vote to call it [unBASIC'd](https://en.wikipedia.org/wiki/BASIC).